include/seqan3/alignment/configuration/align_config_output.hpp

```diff
@@ -283,6 +262,22 @@ struct output_sequence2_id_tag : public pipeable_config_element<output_sequence2
  * \see seqan3::align_cfg::output_alignment
  * \see seqan3::align_cfg::output_sequence1_id
  */
-inline constexpr output_sequence2_id_tag output_sequence2_id{};
+struct output_sequence2_id : public pipeable_config_element<output_sequence2_id>
+{
+    /*!\name Constructor, destructor and assignment
+     * \{
+     */
+    constexpr output_sequence2_id() = default; //!< Defaulted.
+    constexpr output_sequence2_id(output_sequence2_id const &) = default; //!< Defaulted.
+    constexpr output_sequence2_id(output_sequence2_id &&) = default; //!< Defaulted.
+    constexpr output_sequence2_id & operator=(output_sequence2_id const &) = default; //!< Defaulted.
+    constexpr output_sequence2_id & operator=(output_sequence2_id &&) = default; //!< Defaulted.
+    ~output_sequence2_id() = default; //!< Defaulted.
+    //!\}
+
```

Comment:
Most other config elements which do not have any member variable do not have explicit ctors
```suggestion
```